google/cloud/forseti/services/base/config.py

```diff
@@ -148,15 +149,50 @@ def get_storage_class(self):
 
 
 class InventoryConfig(AbstractInventoryConfig):
+    # pylint: disable=too-many-instance-attributes
     """Implements composed dependency injection for the inventory."""
 
+    @staticmethod
+    def _filter_valid_resources(excluded_resources):
```

Comment:
Hmmm, the naming of this method is not quite clear to me, especially when looking at the code.  It seems like you are validating the given excluded resources.  So, I would call this method `_validate_excluded_resources()`

Reply:
Or, `filter_invalid_excluded_resources()`.

Reply:
I have already put the explanation in the docstring, method name cannot be that long so both of the names above will create a pylint failure. The name should be fairly clear already as we are filtering all the valid resources from a list of excluded resources.